Python/bytecodes.c

```diff
@@ -1339,29 +1366,39 @@ dummy_func(
             Py_DECREF(oldobj);
         }
 
-        inst(LOAD_CLASSDEREF, ( -- value)) {
-            PyObject *name, *locals = LOCALS();
-            assert(locals);
+        op(_LOAD_CLASSDEREF_INTRO, (-- class_dict)) {
```

Comment:
No need for this if you change the instructions as described above. This is `LOAD_LOCALS_DICT`

Reply:
This is gone.